packages/react-scripts/fixtures/kitchensink/integration/env.test.js

```diff
@@ -6,7 +6,15 @@ describe('Integration', () => {
     it('file env variables', async () => {
       const doc = await initDOM('file-env-variables')
 
-      expect(doc.getElementById('feature-file-env-variables').textContent).to.equal('fromtheenvfile.')
```

Comment:
@EnoahNetzach was there a reason you included the periods? I don't want to remove them if they were there for a reason.